index.js

```diff
@@ -6,14 +6,16 @@
 
 module.exports = function(ServerlessPlugin) {
 
-  const path       = require('path'),
-      fs           = require('fs'),
-      os           = require('os'),
-      babelify     = require('babelify'),
-      browserify   = require('browserify'),
-      UglifyJS     = require('uglify-js'),
-      wrench       = require('wrench'),
-      BbPromise    = require('bluebird');
+const _              = require('lodash'),
+      BbPromise      = require('bluebird'),
+      UglifyJS       = require('uglify-js'),
```

Comment:
Could we make it even more generic by removing the `minify` support and let developers use the [uglifyfy](https://github.com/hughsk/uglifyify) transform, or any other minifier transform?